electrumsv/network_support/general_api.py

```diff
@@ -1296,7 +1296,7 @@ async def prepare_server_tip_filter_peer_channel(indexing_server_state: ServerCo
     # NOTE(typing) Type is incompatible with same type, who knows? Error message as follows:
     # `Argument 1 to "update" of "TypedDict" has incompatible type "IndexerServerSettings";
     # expected "TypedDict({'tipFilterCallbackUrl'?: str | None})"  [typeddict-item]`
-    indexing_server_state.indexer_settings.update(settings_delta_object) # type: ignore
+    indexing_server_state.indexer_settings.update(settings_delta_object)
```

Comment:
The comment is there to detail why we were overriding the type checking. Did you read it? Is there a reason why you wouldn't remove that comment if you remove the type checking override? I do not understand.

Reply:
I didn't see the comment above it actually. Will remove it.